cmd/cnab-to-oci/main.go

```diff
@@ -3,14 +3,25 @@ package main
 import (
 	"os"
 
+	"github.com/sirupsen/logrus"
 	"github.com/spf13/cobra"
 )
 
 func main() {
+	var logLevel string
 	cmd := &cobra.Command{
 		Use:          "cnab-to-oci <subcommand> [options]",
 		SilenceUsage: true,
+		PersistentPreRunE: func(cmd *cobra.Command, args []string) error {
+			level, err := logrus.ParseLevel(logLevel)
+			if err != nil {
+				return err
+			}
+			logrus.SetLevel(level)
+			return nil
+		},
 	}
+	cmd.PersistentFlags().StringVar(&logLevel, "log-level", "warning", "set the log-level {trace|debug|info|warning|error|fatal|panic}")
```

Comment:
could we make this consistent with the docker cli: `Set the logging level ("debug"|"info"|"warn"|"error"|"fatal") (default "info")`

Reply:
Do you mean you want the default log level to `info` too?

Reply:
Yes

Reply:
If the default log level is `info` I think that most (all?) of log statements should be in `debug`. I think that a library must be quiet with the default log level.